ui/app/components/clients/current.js

```diff
@@ -29,71 +29,38 @@ export default class Current extends Component {
       let findUpgrade = versionHistory.find((versionData) => versionData.id.match(version));
       if (findUpgrade) relevantUpgrades.push(findUpgrade);
     });
-
-    // if no history for 1.9 or 1.10, customer skipped these releases so get first stored upgrade
-    // TODO account for customer STARTING on 1.11
-    if (relevantUpgrades.length === 0) {
-      relevantUpgrades.push({
-        id: versionHistory[0].id,
-        previousVersion: versionHistory[0].previousVersion,
-        timestampInstalled: versionHistory[0].timestampInstalled,
-      });
-    }
     // array of upgrade data objects for noteworthy upgrades
     return relevantUpgrades;
   }
 
-  // Response total client count data by namespace for current/partial month
-  get byNamespaceTotalClients() {
-    return this.args.model.monthly?.byNamespaceTotalClients || [];
-  }
-
-  // Response new client count data by namespace for current/partial month
-  get byNamespaceNewClients() {
-    return this.args.model.monthly?.byNamespaceNewClients || [];
+  // Response client count data by namespace for current/partial month
+  get byNamespace() {
+    return this.args.model.monthly?.byNamespace || [];
   }
 
   get isGatheringData() {
     // return true if tracking IS enabled but no data collected yet
-    return (
-      this.args.model.config?.enabled === 'On' &&
-      this.byNamespaceTotalClients.length === 0 &&
-      this.byNamespaceNewClients.length === 0
-    );
+    return this.args.model.config?.enabled === 'On' && this.byNamespace.length === 0;
   }
 
   get hasAttributionData() {
     if (this.selectedAuthMethod) return false;
     if (this.selectedNamespace) {
       return this.authMethodOptions.length > 0;
     }
-    return this.totalUsageCounts.clients !== 0 && !!this.totalClientsData;
+    return this.totalUsageCounts.clients !== 0 && !!this.totalClientAttribution;
   }
 
-  get filteredTotalData() {
+  get filteredCurrentData() {
     const namespace = this.selectedNamespace;
     const auth = this.selectedAuthMethod;
     if (!namespace && !auth) {
-      return this.byNamespaceTotalClients;
+      return this.byNamespace;
     }
     if (!auth) {
-      return this.byNamespaceTotalClients.find((ns) => ns.label === namespace);
+      return this.byNamespace.find((ns) => ns.label === namespace);
     }
-    return this.byNamespaceTotalClients
```

Comment:
Removing as no longer showing new vs total client counts in current/partial month view